### PR TITLE
Bump graphiql and add support to v2 configurations

### PR DESCRIPTION
**Pull request by joonaojapalo**
Changes:
 - fix broken build (yarn start) in master by graphiql version bump
 - add v2 router configurations
 - handle use cases eg. when user changes API version to production but such router eg. prodiction/hsl v2 doesn't exist

Related issue: DT-5292

### CODE REVIEW

**Review by optionsome — CHANGES_REQUESTED** [review]
Typing something in the query seems to cause rerender and new requests to be sent which also affects how the autocomplete works.

**Comment by joonaojapalo** [comment]
Autocomplete flickering issue solved.

File: src/App.js [hunk 1]
```diff
@@ -3,17 +3,106 @@ import { BrowserRouter as Router } from 'react-router-dom';
...
+
 const configs = [
-  { title: 'HSL', router: 'hsl' },
-  { title: 'Waltti', router: 'waltti' },
-  { title: 'Finland', router: 'finland' }
+  {
+    router: 'hsl',
+    api: {
+      v1: {
+        title: 'HSL (otp1)',
```
Comment (optionsome) [inline src/App.js:26]:
```suggestion
        title: 'HSL (v1)',
```

File: src/App.js [hunk 2]
```diff
@@ -3,17 +3,106 @@ import { BrowserRouter as Router } from 'react-router-dom';
...
+        routerUrl: {
+          dev: buildUrl('dev', VERSION_1, 'hsl'),
+          prod: buildUrl('prod', VERSION_1, 'hsl')
+        }
+      },
+      v2: {
+        title: 'HSL (otp2)',
```
Comment (optionsome) [inline src/App.js:33]:
```suggestion
        title: 'HSL (v2)',
```

File: src/App.js [hunk 3]
```diff
@@ -3,17 +3,106 @@ import { BrowserRouter as Router } from 'react-router-dom';
...
+          dev: buildUrl('dev', VERSION_2, 'hsl')
+        }
+      }
+    }
+  },
+  {
+    router: 'waltti',
+    api: {
+      v1: {
+        title: 'Waltti (otp1)',
```
Comment (optionsome) [inline src/App.js:44]:
```suggestion
        title: 'Waltti (v1)',
```

File: src/App.js [hunk 4]
```diff
@@ -3,17 +3,106 @@ import { BrowserRouter as Router } from 'react-router-dom';
...
+        routerUrl: {
+          dev: buildUrl('dev', VERSION_1, 'waltti'),
+          prod: buildUrl('prod', VERSION_1, 'waltti')
+        }
+      },
+      v2: {
+        title: 'Waltti (otp2)',
```
Comment (optionsome) [inline src/App.js:51]:
```suggestion
        title: 'Waltti (v2)',
```

File: src/App.js [hunk 5]
```diff
@@ -3,17 +3,106 @@ import { BrowserRouter as Router } from 'react-router-dom';
...
+          dev: buildUrl('dev', VERSION_2, 'waltti')
+        }
+      }
+    }
+  },
+  {
+    router: 'waltti-alt',
+    api: {
+      v2: {
+        title: 'Waltti alt. (otp2)',
```
Comment (optionsome) [inline src/App.js:62]:
```suggestion
        title: 'Waltti alt. (v2)',
```

File: src/App.js [hunk 6]
```diff
@@ -3,17 +3,106 @@ import { BrowserRouter as Router } from 'react-router-dom';
...
+          prod: buildUrl('prod', VERSION_2, 'waltti-alt')
+        }
+      }
+    }
+  },
+  {
+    router: 'finland',
+    api: {
+      v1: {
+        title: 'Finland (otp1)',
```
Comment (optionsome) [inline src/App.js:74]:
```suggestion
        title: 'Finland (v1)',
```

File: src/App.js [hunk 7]
```diff
@@ -3,17 +3,106 @@ import { BrowserRouter as Router } from 'react-router-dom';
...
+        routerUrl: {
+          dev: buildUrl('dev', VERSION_1, 'finland'),
+          prod: buildUrl('prod', VERSION_1, 'finland')
+        }
+      },
+      v2: {
+        title: 'Finland (otp2)',
```
Comment (optionsome) [inline src/App.js:81]:
```suggestion
        title: 'Finland (v2)',
```

File: src/App.js [hunk 8]
```diff
@@ -3,17 +3,106 @@ import { BrowserRouter as Router } from 'react-router-dom';
...
+          dev: buildUrl('dev', VERSION_2, 'finland')
+        }
+      }
+    }
+  },
+  {
+    router: 'varely',
+    api: {
+      v2: {
+        title: 'VARELY (otp2)',
```
Comment (optionsome) [inline src/App.js:92]:
```suggestion
        title: 'VARELY (v2)',
```

**Review by optionsome — COMMENTED** [review]
I think it's better to call the APIs v1/v2 instead of otp1, otp2 because I think we will end up with a v3 that also uses otp2.

File: src/App.js (hunk 1 above)
Comment (optionsome) [inline src/App.js:26]:
```suggestion
        title: 'HSL (v2)',
```

**Review by optionsome — APPROVED** [review]

**Pull request merged**